private/js2ts/index.mjs

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
```

Comment:
what's this script for? cannot see it referenced anywhere

Reply:
To convert package from JS to TS (hence the name)